client/src/Pages/Home/index.tsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react'
+import Characters from 'Components/Characters'
 // ok
-const Home: React.FC = () => <div>home</div>
+const Home: React.FC = () => <div>
+    <Characters />
```

Comment:
Maybe create another page for the characters, since the homepage will not be the character page

Reply:
Wait just saw that this is not your work, nvm

Reply:
Yeah I branched off @Burbukje work because she already has all the characters